ui/pages/confirmations/components/stx-banner-alert/stx-banner-alert.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+import {
+  BannerAlert,
+  ButtonLink,
+  Text,
+  BannerAlertSeverity,
+} from '../../../../components/component-library';
+import {
+  stxAlertIsOpen,
+  dismissSTXMigrationAlert,
+} from '../../../../ducks/alerts/stx-migration';
+import ZENDESK_URLS from '../../../../helpers/constants/zendesk-url';
+
+const STXBannerAlert = () => {
+  const dispatch = useDispatch();
+  const shouldShow = useSelector(stxAlertIsOpen);
```

Comment:
Can you rename `stxAlertIsOpen` to `shouldShowSmartTransactionsMigrationAlert` to make it little clearer?
```
 const showSmartTransactionsMigrationAlert = useSelector(shouldShowSmartTransactionsMigrationAlert);
```

Alternatively, you can also do this renaming:
```
 const smartTransactionsMigrationAlertSeen = useSelector(wasSmartTransactionsMigrationAlertSeen);

if (smartTransactionsMigrationAlertSeen) {
    return null;
  }
```